migrations/20190110193654_instructions_table.js

```diff
@@ -0,0 +1,17 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable("instructions", instructions=>{
+        instructions.increments();
+        instructions.string('name',255).notNullable();
+        instructions
+        .integer("recipe_id")
+        .unsigned()
+        .references("id")
+        .inTable("recipes")
+        .notNullable();
+    })
+};
+  
+   exports.down = function(knex, Promise) {
+    return knex.schema.dropTableIfExists('intructions')
```

Comment:
It looks like you have an _s_ missing here. One thing that I've seen to avoid this is actually using a variable. For example:
```javascript
let instructions = 'instructions';

exports.up = function(knex, Promise) {
    return knex.schema.createTable(instructions, instructions=>{
        instructions.increments();
        instructions.string('name',255).notNullable();
        instructions
        .integer("recipe_id")
        .unsigned()
        .references("id")
        .inTable("recipes")
        .notNullable();
    })
};

exports.down = function(knex, Promise) {
    return knex.schema.dropTableIfExists(instructions)
};
```
This way, the createTable and dropTable will both always be consistent.